web/package.json

```diff
@@ -11,10 +11,12 @@
     "build-storybook": "build-storybook",
     "test": "vitest run",
     "test-watch": "vitest",
-    "coverage": "vitest run --coverage"
+    "coverage": "vitest run --coverage",
+    "playwright": "npx playwright test"
```

Comment:
No stong opinion, but probably run using npm as we install it as dev dependency?